Simplenote/src/main/java/com/automattic/simplenote/authentication/SessionManager.kt

```diff
@@ -0,0 +1,21 @@
+package com.automattic.simplenote.authentication
+
+import com.simperium.Simperium
+import com.simperium.client.User
+import javax.inject.Inject
+
+sealed class UserSession {
+    object UnauthorizedUser : UserSession()
+    data class AuthorizedUser(val user: User) : UserSession()
+}
+
+class SessionManager @Inject constructor(private val simperium: Simperium) {
```

Comment:
Question (❓): Is it possible to add a test suite to accompany this `SessionManager` class?

Reply:
Sure. Added in c0c9db1a3608c9121085311ce1fb3f661c0c13eb.

Reply:
Awesome @danilo04 , many thanks and it lots good! 🚀